test/functional/api/search_api_test.rb

```diff
@@ -163,19 +167,24 @@ def app
     assert_equal "/profile/steff2",     json['items'][0]['doc_url']
     assert_equal "/profile/steff3",     json['items'][1]['doc_url']
 
+    puts matcher.inspect
```

Comment:
Couple comments left over here!

This looks great. So if I read right, this uses okonek's tagname (lat:0000) string parsing approach to bounding boxes while nearby nodes uses the new decimal coordinate columns?

Reply:
This uses tagname, nearby nodes needs to be refactor to use the new decimal coordinate columns. I'm still need to implement the new decimal coordinate deletion and then I can open a new issue to refactor nearby nodes.

Reply:
And I removed the ‘puts’!